### ISSUE TITLE
When adding external evaluators to evaluation, admin needs to specify for which project the evaluation is for

### ISSUE THREAD

**Issue by sheilaramas** [nerubia/kh360-react#203]
**DESCRIPTION**

- [x] When adding external evaluators, project member should be specified. 
- [x] This only applies to templates where both evaluee and evaluator id's are not null
- [x] For those which has evaluee or evaluator id's which are null, no need to link to a project_member record

![Image](https://github.com/nerubia/kh360-react/assets/1640285/4c74da6b-e8b1-4321-893f-3b5b1ea0e1e1)

**Comment by sheilaramas** [comment]
Maybe when adding NEW external evaluator, add a project list at the bottom of the screen



![Image](https://github.com/nerubia/kh360-react/assets/1640285/443862e5-0166-4ef5-aa4d-954379671279)



- [ ] Add table containing list of projects with the following columns:
          - checkbox
          - Project Name
          - Evaluee Role
          - % Assignment
          - Duration

- [ ] So admin has to tick on the list to specify which projects they will add that evaluator to
- [ ] Admin must tick at least 1 project
- [ ] Once saved, go back to Step 4 screen and add the external evaluator... 1 evaluation record per project (so if the admin ticked 2 projects, there must be 2 evaluation records added)

**Comment by sheilaramas** [comment]
**FOR ADD FROM LIST**




![Image](https://github.com/nerubia/kh360-react/assets/1640285/25fdaf55-8025-4b5b-b6a7-a71f22ca9a7a)

**Comment by sheilaramas** [comment]
**UPDATED IMPLEMENTATION**

- [ ] In Step 4, add a column after Evaluee Role
- [ ] This column will contain a dropdown containing options for which project_member record to assign the evaluator to
- [ ] In dropdown box, display project name, percentage assignment and start and end date
- [ ] Project Member record can be selected only once per evaluator (cannot contain evaluators with duplicate project member records associated with it)
- [ ] When `Mark as Ready` is clicked, double check the following:
          - that all external evaluators have project_member records assigned/selected
          - that a project_member record can only be assigned once per evaluator

**Comment by sheilaramas** [comment]
@jatnerubia  please check, I can't see the project button in external evaluators



![Image](https://github.com/nerubia/kh360-react/assets/1640285/75b48847-c5fd-4600-a542-34531326b5c4)

**Comment by jatnerubia** [comment]
PR: https://github.com/nerubia/kh360-react/pull/224

**Comment by sheilaramas** [comment]
Verified all acceptance criteria above. But there is a special scenario that can't be tested with current implementation (having the same evaluators for different projects). Separate ticket to be created to allow being able to select an evaluator more than once) and testing for this scenario will be done on that ticket.
